src/components/Layout/index.tsx

```diff
@@ -3,55 +3,41 @@ import Link from 'next/link';
 import { Box } from '@mui/system';
 import { Button, ButtonGroup } from '@mui/material';
 import useWindowSize from '@/utils/useWindowSize';
-import RenderIf from '@/utils/RenderIf';
-import HamburgerDropDown from '@/components/HambergerDropdown';
+import RenderIf from '@/utils/renderIf';
+import HamburgerDropDown from '@/components/HamburgerDropdown';
 import {
-  colors,
-  mediaQueriesThresholds,
-  headerNavigation
+  MEDIA_QUERIES_THRESHOLDS,
+  HEADER_NAVIGATION,
 } from '@/constants';
 
+import { styles }from './styles'
+
 const Layout = ({ children }: any) => {
   const { width } = useWindowSize();
   return (
     <div>
       <header>
-        <Box sx={{
-          display: 'flex',
-          flexDirection: 'row',
-          color: 'white',
-          backgroundColor: 'grey',
-          opacity: '.8',
-          fontFamily: 'Bungee Hairline'
-        }}>
+        <Box sx={styles.header}>
           <Link href='/'>
               <h1>PDX Bots</h1>
           </Link>
-          <Box sx={{
-              display: 'flex',
-              justifyContent: 'flex-end',
-              width: '100%',
-              alignItems: 'center',
-              color: colors.text,
-              opacity: '1',
-              paddingRight: '2rem'
-            }}>
-            <RenderIf isTrue={width && width > mediaQueriesThresholds.mobile}>
+          <Box sx={styles.headerNavBox}>
+            <RenderIf isTrue={width && width > MEDIA_QUERIES_THRESHOLDS.mobile}>
               <ButtonGroup>
-                {headerNavigation.map((link) => (
+                {HEADER_NAVIGATION.map((link) => (
                   <Link href={link.href} key={link.label}>
-                    <Button 
+                    <Button
                       variant='text'
-                      sx={{ color:'#16FF00', fontFamily: 'Bungee Hairline', fontWeight:400}}>
+                      sx={styles.navigationButton}>
                         {link.label}
                     </Button>
                   </Link>
                 ))}
               </ButtonGroup>
             </RenderIf>
-            <RenderIf isTrue={width && width <= mediaQueriesThresholds.mobile}>
-              <HamburgerDropDown headerNavigation={headerNavigation}/>
-            </RenderIf>           
+            <RenderIf isTrue={width && width <= MEDIA_QUERIES_THRESHOLDS.mobile}>
+              <HamburgerDropDown HEADER_NAVIGATION={HEADER_NAVIGATION}/>
```

Comment:
Yeah you shouldn't prop drill this. It's something that should just be consumed from the source in the drop down.